proxmox/datadog_checks/proxmox/constants.py

```diff
@@ -0,0 +1,16 @@
+# (C) Datadog, Inc. 2025-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+RESOURCE_TYPE_MAP = {
+    'qemu': 'vm',
+    'lxc': 'container',
```

Comment:
```suggestion
    'lxc': 'container',
    'openvz': 'container',
```
nit: I guess it wouldn't hurt to add this, even if openvz is outdated.